simcon_project/templates/template_management/create_conversation_template.html

```diff
@@ -0,0 +1,108 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Creating Conversation Template</title>
+    {% load static %}
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="{% static 'conversation_templates/create_conversation_template.css' %}">
+    <script src="{% static 'conversation_templates/create_conversation_template.js' %}" type="text/javascript"></script>
+</head>
+
+<body>
+
+<!-- TODO:Delete? -->
```

Comment:
should this still be here?

Reply:
Yeah... I might eventually use it (still have not nailed down the exact UI layout there... especially if we extend this page to edit templates.